test/packages/knowledge_base/kibana/kb_entry/foo/index-mapping.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
If all knowledge bases are expected to have the same mappings, I think the mapping could be provided by Fleet. We should include in packages only things that are different between them.